src/review/Post.js

```diff
@@ -0,0 +1,145 @@
+import React from 'react';
+import './Post.scss';
+import { AiOutlineArrowUp, AiOutlineArrowDown } from 'react-icons/ai';
```

Comment:
```suggestion
import { AiOutlineArrowUp, AiOutlineArrowDown } from 'react-icons/ai';
import './Post.scss';
```

scss import를 마지막에 해주세요

Reply:
수정완료!